### PR TITLE
chore: update code examples in readme

### PR DESCRIPTION
**Pull request by ncaudill27**
Update code examples in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -152,27 +154,31 @@ Let's create an index for our _posts_.
...
       terms: [{ field: ["data", "title"] }]
     })
-{ ref: Index("posts_by_title"),
+{
+  ref: Index("posts_by_title"),
   ts: 1532624135128797,
   active: false,
   partitions: 1,
   name: 'posts_by_title',
   source: Class("posts"),
```
Comment (evbruno) [inline README.md:166]: missed that `Class` here

File: README.md [hunk 2]
```diff
@@ -152,27 +154,31 @@ Let's create an index for our _posts_.
...
 Let's insert a _post_ item:
 
 ```javascript
 my_app> Create(
-    Class("posts"),
+    Collection("posts"),
     { data: { title: "What I had for breakfast .." } })
-{ ref: Ref(Class("posts"), "205904004461363712"),
+{
+  ref: Ref(Class("posts"), "205904004461363712"),
```
Comment (evbruno) [inline README.md:178]: 👆🏻

**Comment by ncaudill27** [comment]
I missed all the output references! Fixed it now.

**Comment by evbruno** [comment]
oops... there's the [other session](https://github.com/fauna/fauna-shell/blob/33b61a4935326a4c9457571cfb80177a9865207d/README.md#executing-queries-from-a-file) with deprecated examples

**Comment by ncaudill27** [comment]
Alright double checked and they should all be gone now. There was another on 835.  Sorry about that.

**Comment by evbruno** [comment]
one last thing (not related to deprecated examples), there's an extra quotes on the last session:
https://github.com/fauna/fauna-shell/blob/1028b99b558fada949837c113890bd9f80e966b2/README.md#fauna-eval-query

**Comment by ncaudill27** [comment]
It seemed that it was missing quotes. The commands only worked for me if each argument was wrapped in quotes.

**Comment by ncaudill27** [comment]
Wrapping 2 or 3 arguments together in quotes throws. `Error: Error: Line 1: Invalid left-hand side in assignment`

**Comment by evbruno** [comment]
You can run `fauna eval arg1 arg2`, or `fauna eval "arg1" "arg2"` or even combine them `fauna eval "arg1" arg2`

..but this this PR is good to go. Thanks for taking the time to fix it!  👍

**Comment by ncaudill27** [comment]
Gotcha. Anytime! Having a good time learning how to implement graphql.

**Comment by evbruno** [comment]
oh, there's a "squash and merge" option in the merge button... sorry about that...

this PR is good... closing it :)

**Pull request merged**